src/pages/program-management/tabs/transfer-out-tab.component.tsx

```diff
@@ -43,7 +44,15 @@ const columnsLab: EncounterListColumn[] = [
     key: 'verified',
     header: 'Verified',
     getValue: encounter => {
-      return getObsFromEncounter(encounter, verified_UUID);
+      const obs = findObs(encounter, verified_UUID);
+        if (!obs) {
```

Comment:
@ODORA0 u need to install `es-lint` to ur editor to automatically fix these. It's failing the CI

Reply:
@pirupius Done